sound/soc/sof/sof-audio.h

```diff
@@ -106,7 +106,7 @@ struct snd_sof_route {
 };
 
 /* ASoC DAI device */
-struct snd_sof_dai {
+struct snd_sof_dai_link {
```

Comment:
line 116 comment should be list in sdev dai_link list

I am still in the dark on this, on one side this looks like a dailink but on the other we have the ability to handle a single DAI represented in the firmware (single comp_dai). I am actually at a loss as to how this works for SoundWire, where we have two SoundWire dais in the same dailink when we do speaker aggregation.

Need more coffee on this one.